integrations/tensorflow/test/tflite_integration/mobilenet_v1_test.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2020 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import absl.testing
+import numpy
+import test_util
+
+model_path = "https://storage.googleapis.com/iree-model-artifacts/tflite-integration-tests/mobilenet_v1.tflite"
```

Comment:
I'm concerned about anything requiring fetching from the web being run by default on our CI as part of a run that doesn't fetch from the web. There are obvious advantages to fetching these as part of the test, but network access is a classic source of flakiness in tests.

Reply:
<offline discussion> we are planning to better stage compilation so separate pre-submits can execute without rebuilding the project every time. Then we can revisit good fetching mechanisms for test models. Until then we can enable with a "ignore if failed" policy.

Reply:
Just noting that Rob and I discussed this offline and I don't actually have a suggestion for a better option that is worth the work